content/audio.md

```diff
@@ -17,29 +17,20 @@ section: software-troubleshooting
 tableOfContents: true
 ---
 
-Sound settings or packages related to the sound system can become corrupt or broken. Many times, deleting the configuration files, reinstalling the sound-related packages, and reloading ALSA and/or PulseAudio can help. These commands can also help fix the <u>Sound</u> settings showing "Dummy Output" as the audio output.
+Sound settings or packages related to the sound system can become corrupt or broken. Many times, deleting the configuration files, reinstalling the sound-related packages, and starting the audio software can help. These commands can also help fix the <u>Sound</u> settings showing "Dummy Output" as the audio output.
 
-## Reset PulseAudio
+## Reset Audio Software (server)
 
-If the system is not playing audio, first try restarting the PulseAudio daemon:
+If the system is not playing audio, first try restarting the Audio daemon:
 
-```
-systemctl --user restart pulseaudio
-```
-
-After restarting the daemon, applications may need to be restarted to re-connect to PulseAudio. If the system still isn't playing sound, then try removing the user configuration files for PulseAudio:
-
-```
-rm -r ~/.config/pulse
-```
-
-Then, kill all instances of PulseAudio:
+| PulseAudio | PipeWire |
+| :--------- | :-------|
+| Ubuntu and Pop!\_OS pre-22.04 | Pop!\_OS 22.04+ |
+| `systemctl --user restart pulseaudio` | `systemctl --user restart restart wireplumber pipewire pipewire-pulse` |
+| `rm -r ~/.config/pulse` |  `rm -r ~/.config/pulse` |
+| `pulseaudio -k` | |
 
-```
-pulseaudio -k
-```
-
-When PulseAudio starts again (which it should do automatically), it will create new configuration files.
+This set of commands first restarts the sound daemon and removes the users configuration for PulseAudio. If still using PulseAudio as a server, restarts the PulseAudio server that will create new default audio configuration files.
```

Comment:
In the first sentence, `users` should have an apostrophe.

In the second sentence, you're missing a subject in the clause after the comma.